src/nxdoc/generic-multi-excerpts.md

```diff
@@ -142,3 +142,21 @@ This requires to have the JSF UI addon installed on your server that is deprecat
 Please refer to the [Web UI documentation](https://doc.nuxeo.com/nxdoc/web-ui/).
 {{/callout}}
 {{! /multiexcerpt}}
+
+{{! multiexcerpt name='lts2021-general-prerequesites'}}
+
+You need to check the following items in order to install Nuxeo Platform LTS 2021:
+- Get a Nuxeo Online Services account. If not, please check the [Nuxeo Download page](https://www.nuxeo.com/downloads/) and create a trial account.
```

Comment:
Does this make sense in the general installation prerequisites as it isn't needed to pull the Docker image?

Reply:
As viewed with @aescaffre , we should consider people reading the doc are clients. And we don't want to focus on people who just pull the default image without installing any addon which requires no subscription. So, if I'm a Nuxeo partner, or a client, I need this information if I need to do something useful. Maybe we can just mention that these elements are not needed to simply pull the image?

Reply:
OK makes sense. Yes, that would be more precise.